playPage.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <!-- the CSS file --> 
+    <link rel="stylesheet" href="style.css" />
+    <title>YouTube clone with HTML & CSS</title>
+</head>
+
+
+<body>
```

Comment:
For readability, I suggest that you should add two spaces of indentation. Do not use the tab key.